common/lib/xmodule/xmodule/xml_module.py

```diff
@@ -293,8 +313,12 @@ def from_xml(cls, xml_data, system, id_generator):
             definition_xml = cls.load_file(filepath, system.resources_fs, def_id)
             system.parse_asides(definition_xml, def_id, usage_id, id_generator)
         else:
-            definition_xml = xml_object
             filepath = None
+            definition_xml = xml_object
+            dog_stats_api.increment(
+                'vscompat.deprecation',
+                tags=("location:xmlparser_util_mixin_parse_xml")
```

Comment:
You need a trailing comma here to make it a singleton tuple `("...", )`, or you need to use a list `["..."]`.